include/index/make_index.h

```diff
@@ -89,7 +89,7 @@ std::shared_ptr<Index> make_index(const cpptoml::table& config, Args&&... args)
         config, std::forward<Args>(args)...);
 
     // if index has already been made, load it
-    if (filesystem::make_directory(idx->index_name()) && idx->valid())
+    if (!filesystem::make_directory(idx->index_name()) && idx->valid())
```

Comment:
I think so. Probably introduced with the `filesystem` refactoring. `filesystem::make_directory` returns true if it actually *did* make the directory, and false otherwise (e.g. it already exists).